testutils/clustermanager/util_test.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clustermanager
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestStandardExec(t *testing.T) {
+	datas := []struct {
+		cmd    string
+		args   []string
+		expOut string
+		expErr error
+	}{
+		{"bash", []string{"-c", "echo"}, "\n", nil},
```

Comment:
It's more robust to make `echo` display something. 

```suggestion
		{"bash", []string{"-c", "echo", "foo"}, "foo\n", nil},
```

Reply:
done